### PR TITLE
Add wslay 1.1.1

### PR DESCRIPTION
**Pull request by intelligide**
Specify library name and version:  **wslay/1.1.1**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by jgsogo — DISMISSED** [review]

File: recipes/wslay/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import os
+import glob
+from conans import ConanFile, tools, CMake
+
+
+class WslayConan(ConanFile):
+    name = "wslay"
+    description = "The WebSocket library in C"
+    topics = ("conan", "websockets")
+    homepage = "https://github.com/tatsuhiro-t/wslay"
```
Comment (uilianries) [inline recipes/wslay/all/conanfile.py:10]:
```suggestion
    homepage = "https://tatsuhiro-t.github.io/wslay"
```

File: recipes/wslay/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
+import os
+import glob
+from conans import ConanFile, tools, CMake
+
+
+class WslayConan(ConanFile):
+    name = "wslay"
+    description = "The WebSocket library in C"
+    topics = ("conan", "websockets")
```
Comment (uilianries) [inline recipes/wslay/all/conanfile.py:9]:
```suggestion
    topics = ("conan", "websockets", "rfc6455", "communication", "tcp")
```

File: recipes/wslay/all/conanfile.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
+import os
+import glob
```
Comment (uilianries) [inline recipes/wslay/all/conanfile.py:2]:
```suggestion
```

Not used

File: recipes/wslay/all/patches/0001-msvc-support.patch [hunk 4]
```diff
@@ -0,0 +1,74 @@
+diff --git a/.gitignore b/.gitignore
```
Comment (uilianries) [inline recipes/wslay/all/patches/0001-msvc-support.patch:1]: That's a great patch, you could open a PR to the author.

**Review by uilianries — CHANGES_REQUESTED** [review]
Thanks for your contribution again! Almost good to go!

File: recipes/wslay/all/patches/0001-msvc-support.patch (hunk 4 above)
Comment (intelligide) [reply recipes/wslay/all/patches/0001-msvc-support.patch:1]: It's already done, I just used the differential of this PR https://github.com/tatsuhiro-t/wslay/pull/36/files.

Comment (uilianries) [reply recipes/wslay/all/patches/0001-msvc-support.patch:1]: Thank you!

**Review by uilianries — APPROVED** [review]
LGTM

**Review by jgsogo — APPROVED** [review]

**Review by prince-chrismc — APPROVED** [review]

**Pull request merged**